rpm/publish/gen.rb

```diff
@@ -21,8 +22,8 @@
 To use this repository, run the following command:
 
 <pre style="padding:0.5em; margin:1em; background:black; color:white">
-sudo wget -O /etc/yum.repos.d/jenkins.repo #{url}/#{artifactName}.repo
-sudo rpm --import #{url}/jenkins-ci.org.key
+sudo wget -O /etc/yum.repos.d/#{artifactName}.repo #{url}/#{artifactName}.repo
```

Comment:
The actual repo file does not use artifact names in the repo name, in docs for installation, even for enterprise distribution.

This needs to be in a separate PR, because we're actually changing behavior here and documentation needs to change at same time.